src/daemon/coordinator.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <daemon/base_daemon.h>
+
+namespace dist_clang {
+namespace daemon {
+
+class Coordinator : public BaseDaemon {
+ public:
+  explicit Coordinator(const proto::Configuration& configuration);
+
+  bool Initialize() override;
+  inline bool UpdateConfiguration(
+      const proto::Configuration& configuration) override {
+    return true;
+  }
+
+ private:
+  bool HandleNewMessage(net::ConnectionPtr connection, Universal message,
+                        const net::proto::Status& status) override;
+
+  const proto::Host local_;
```

Comment:
Why not to use `SharedPtr<const proto::Configuration> conf_;` like in `CompilationDaemon`? - It's potentially easier this way to reload config from local file system. Also, you preserve the config structure only in one place - the `.proto` file.